### PR TITLE
Update .vsts-ci.yml for Azure Pipelines, remove Win2016 build

### PR DESCRIPTION
**Pull request by ericwol-msft**

### CODE REVIEW

File: build_all/windows/build.cmd [hunk 1]
```diff
@@ -97,6 +97,14 @@ rem no error checking
 
 pushd %build-root%\cmake\%CMAKE_DIR%
 
+echo ***checking msbuild***
+where /q msbuild
+IF ERRORLEVEL 1 (
+echo ***setting VC paths***
+    IF EXIST "%ProgramFiles(x86)%\Microsoft Visual Studio\2017\Enterprise\Common7\Tools\VsMSBuildCmd.bat" call "%ProgramFiles(x86)%\Microsoft Visual Studio\2017\Enterprise\Common7\Tools\VsMSBuildCmd.bat"
```
Comment (danewalton) [inline build_all/windows/build.cmd:104]: Should this be C:\Program Files as above in the YAML

File: build_all/windows/build.cmd [hunk 2]
```diff
@@ -97,6 +97,14 @@ rem no error checking
...
+)
+where msbuild
+
```
Comment (danewalton) [inline build_all/windows/build.cmd:107]: Error check to make sure like line 102?

**Review by danewalton — APPROVED** [review]

File: build_all/windows/build.cmd (hunk 1 above)
Comment (ericwol-msft) [reply build_all/windows/build.cmd:104]: done

File: build_all/windows/build.cmd [hunk 3]
```diff
@@ -97,6 +97,19 @@ rem no error checking
...
+echo ***checking msbuild***
+where /q msbuild
+IF ERRORLEVEL 1 (
+echo ***setting VC paths***
+    IF EXIST "%ProgramFiles(x86)%\Microsoft Visual Studio\2017\Enterprise\Common7\Tools\VsMSBuildCmd.bat" call "%ProgramFiles(x86)%\Microsoft Visual Studio\2017\Enterprise\Common7\Tools\VsMSBuildCmd.bat"
+)
+
+where msbuild
+IF ERRORLEVEL 1 (
+    echo [91mERROR: msbuild not found!!![0m
```
Comment (danewalton) [inline build_all/windows/build.cmd:109]: are these non-ascii characters?

Comment (ericwol-msft) [reply build_all/windows/build.cmd:109]: yes, this are ESC chars which will cause the text to print in RED.

**Pull request merged**